src/module/PrcaOrchestrator.ts

```diff
@@ -111,4 +116,15 @@ export class PrcaOrchestrator {
         return result;
     }
 
+    /* Static methods */
+
+    public static CreatePrcaOrchestrator(logger: ILogger, collectionUrl: string, token: string, repositoryId: string, pullRequestId: number): PrcaOrchestrator {
```

Comment:
I don't agree with this. This method does more than construct PrcaOrchestrator, it also instantiates some other classes. Therefore it wouldn't make sense to move the current constructor to a static helper method, because we would have to then bypass the creation of IPrcaService, ISonarQubeReportProcessor, etc in order to use it in test.